src/Hedgehog/Extras/Test/Base.hs

```diff
@@ -156,6 +157,25 @@ failWithCustom cs mdiff msg = liftTest $ mkTest (Left $ H.Failure (getCaller cs)
 failMessage :: MonadTest m => CallStack -> String -> m a
 failMessage cs = failWithCustom cs Nothing
 
+-- | Invert the behavior of a property: success becomes failure and vice versa.
+expectFailure :: (MonadTest m, MonadIO m, HasCallStack) => H.TestT IO a -> m ()
+expectFailure prop = GHC.withFrozenCallStack $ do
```

Comment:
well, you can define `expectFailure` in terms of `expectFailureWith` now 😃 
```haskell
expectFailure = expectFailureWith $ prop ()
```

Reply:
You are right, I don't know how I missed that

Reply:
ah and it needs `GHC.withFrozenCallStack` as well - I keep forgetting about that.

Reply:
Addressed here: https://github.com/input-output-hk/hedgehog-extras/pull/72/commits/6b40d2dfc2eb84891d19dd0180a1148128cbf864